integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkEnvIT.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.integration.test.spark;
+
+import com.datastrato.gravitino.Catalog;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.client.GravitinoMetaLake;
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import com.datastrato.gravitino.integration.test.container.HiveContainer;
+import com.datastrato.gravitino.integration.test.util.spark.SparkUtilIT;
+import com.datastrato.gravitino.spark.GravitinoSparkConfig;
+import com.datastrato.gravitino.spark.plugin.GravitinoSparkPlugin;
+import com.google.common.collect.Maps;
+import java.util.Collections;
+import java.util.Map;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Setup Hive, Gravitino, Spark, Metalake environment to execute SparkSQL. */
+public class SparkEnvIT extends SparkUtilIT {
+  private static final Logger LOG = LoggerFactory.getLogger(SparkEnvIT.class);
+  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+
+  protected final String hiveCatalogName = "hive";
+  private final String metalakeName = "test";
+
+  private SparkSession sparkSession;
+  private String hiveMetastoreUri;
+  private String gravitinoUri;
+
+  @Override
+  protected SparkSession getSparkSession() {
+    Assertions.assertNotNull(sparkSession);
+    return sparkSession;
+  }
+
+  @BeforeAll
+  void startUp() {
+    initHiveEnv();
+    initGravitinoEnv();
+    initMetalakeCatalogs();
```

Comment:
According to the method content, we'd better rename it to `initHiveCatalogs` or something related to catalog hive

Reply:
It will create other catalogs when supporting JDBC or Iceberg, so I prefer not to rename it to hive-related.

Reply:
I prefer to rename to `initMetalakeAndCatalogs`, what do you think?